### PR TITLE
Add content to the Command Registration page

### PR DESCRIPTION
**Pull request by Maenni025**
Added an example command to the Command Registration page and added some explanation.

### CODE REVIEW

File: source/plugins/commands.rst [hunk 1]
```diff
@@ -2,3 +2,76 @@
...
+Now we can register the class in the CommandService
+
+.. code-block:: java
+
+    CommandService cmdService = game.getCommandDispatcher();
+    cmdService.register(plugin, new ExampleCommand(server), "message", "broadcast");
+
+The command is now registered.
+The last argument are the aliases for the command. You can add as many Strings as you want.
+If a alias is used by another command it is ignored. The first alias, which isn't used by another command, becomes the primary alias.
```
Comment (12AwsomeMan34) [inline source/plugins/commands.rst]: an alias*

File: source/plugins/commands.rst [hunk 2]
```diff
@@ -2,3 +2,76 @@
...
+    public class ExampleCommand implements CommandCallable {
+
+        private final Server server;
+
+        public ExampleCommand(Server server) {
+            this.server = server;
+        }
+
+        public boolean call(CommandSource source, String arguments, List<String> parents) throws CommandException {
+            for (Player p : server.getOnlinePlayers()) {
```
Comment (boformer) [inline source/plugins/commands.rst]: use server.broadcastMessage(Message message) instead?

File: source/plugins/commands.rst [hunk 3]
```diff
@@ -2,3 +2,76 @@
...
+
+    import org.spongepowered.api.Server;
+    import org.spongepowered.api.entity.player.Player;
+    import org.spongepowered.api.util.command.CommandCallable;
+    import org.spongepowered.api.util.command.CommandException;
+    import org.spongepowered.api.util.command.CommandSource;
+
+    import com.google.common.base.Optional;
+
+    public class ExampleCommand implements CommandCallable {
```
Comment (boformer) [inline source/plugins/commands.rst]: Rename to MyBroadcastCommand?

File: source/plugins/commands.rst [hunk 4]
```diff
@@ -2,3 +2,76 @@
...
+            }
+            return true;
+        }
+
+        public boolean testPermission(CommandSource source) {
+            return source.hasPermission("example.exampleCommand");
+        }
+
+        public Optional<String> getShortDescription() {
+            Optional<String> desc = Optional.of("Displays a message to all players");
```
Comment (boformer) [inline source/plugins/commands.rst]: You could return these directly, or store them in a field, so you don't have to initialize a new Optional every time.

**Pull request merged**